crates/ruff/src/rules/flake8_copyright/rules/copyright_header_absent.rs

```diff
@@ -0,0 +1,69 @@
+use ruff_diagnostics::Violation;
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::newlines::Line;
+
+use crate::settings::Settings;
+
+use lazy_regex::Regex;
```

Comment:
Nit: if we do need a lazy regex, lets just use `Lazy::new(|| Regex::new(...))`, to avoid adding another dependency.